### PR TITLE
Add new Batch and Student in it

### PR DESCRIPTION
**Pull request by Sejal-gupta-1096**
I have added a new batch and my card to it . Please merge with master branch.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -78,7 +106,7 @@ <h5>October, 2019</h5>
                                 <p>
                                     <small>21 October, 2019</small>
                                 </p>
-                                <a href="./batches/2019/career-camp-21-october-2019/index.html"
+                                <a href="./batches/2020/full-stack-web-dev-2020/index.html"
```
Comment (dakshkhetan) [inline index.html:109]: Undo this change in line 109.

**Review by dakshkhetan — CHANGES_REQUESTED** [review]

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
Any updates on this? This PR will be closed in 24 hours if no changes are made.

**Comment by Sejal-gupta-1096** [comment]
Yes pls.. Give me some time. By today evening i will go through it

File: index.html (hunk 1 above)
Comment (Sejal-gupta-1096) [reply index.html:109]: hey pls tell me in brief what changes are there to be made?

Comment (dakshkhetan) [reply index.html:109]: Sorry for the late response. Merging your branch. :)

**Review by dakshkhetan — APPROVED** [review]
Merging your branch.
